README.md

```diff
@@ -1,4 +1,34 @@
-This repository contains common definitions for the bioconda build system, e.g., the bioconda-utils tag to use and the miniconda version.
+# Overview
 
+This repository acts as a central "source of truth" which can be used by
+various components of the Bioconda build system.
 
-- `common.sh` contains environment variables that control versions
+# Components
+
+**`common.sh`:** contains environment variables that control versions.
+
+**`install-and-set-up-conda.sh`:**
+- downloads and installs conda using the Mambaforge distribution
+- installs bioconda-utils dependencies
+- installs the version of bioconda-utils specified in common.sh via pip install
+
+**`configure-conda.sh`:** ensures channels are configured correctly.
+
+# Typical usage
+
+In practice, a typical CI environment will do the following:
+
+1. Download the files, typically via:
+
+    ```bash
+    wget https://raw.githubusercontent.com/bioconda/bioconda-common/master/{common,install-and-set-up-conda,configure-conda}.sh
```

Comment:
Maybe 3 separate commands? I guess the intention here is to copy-and-paste?

Reply:
I was documenting [what we currently have](https://github.com/bioconda/bioconda-recipes/blob/9223e51cf7c7a697109c6ef1f9643493bcf1542b/azure-pipeline.yml#L23). Could go either way, but I guess it's slightly easier to copy-paste this single line.

Reply:
Ok. Agree.